lib/bolt/config/options.rb

```diff
@@ -501,26 +501,17 @@ module Options
         }
       }.freeze
 
-      # Options that are available in a bolt.yaml file
-      BOLT_OPTIONS = %w[
-        apply-settings
-        color
+      # Options that are available on the command line
+      CLI_OPTIONS = %w[
         compile-concurrency
         concurrency
         format
-        hiera-config
         log
-        modulepath
```

Comment:
`modulepath` can be specified on the CLI - I know it gets converted to a string no matter what, so is valid, but it seems weird to leave out here.